dhcpv4/nclient4/lease.go

```diff
@@ -65,8 +65,8 @@ func (c *Client) Renew(ctx context.Context, lease *Lease, modifiers ...dhcpv4.Mo
 	}
 	if response.MessageType() == dhcpv4.MessageTypeNak {
 		return &ErrNak{
-			Offer: lease.Offer,
-			Nak:   response,
+			Request: request,
```

Comment:
Unless I am missing something, to me it only makes sense to store what the server returns.

Reply:
See https://github.com/insomniacslk/dhcp/pull/470#discussion_r967883390.

Reply:
`Offer` has now been removed.